Makefile

```diff
@@ -88,6 +88,7 @@ mod:
 	@go mod tidy
 	@echo "--> Updating go.mod in ./test/interchain"
 	@(cd ./test/interchain && go mod tidy)
+	@(cd ./test/docker-e2e && go mod tidy)
```

Comment:
```suggestion
	@echo "--> Updating go.mod in ./test/docker-e2e"
	@(cd ./test/docker-e2e && go mod tidy)
```

Reply:
we could replace it with a dynamic one: 

```
mod:
	@echo "Running go mod tidy in all modules"
	@find . -type f -name 'go.mod' -execdir go mod tidy \;
```

Reply:
nice, updated to the dynamic one

Reply:
I'm still seeing the undynamic one so I think a commit hasn't been pushed.

Reply:
apologies, I had two many clones open in parallel and something got lost

should be fixed in https://github.com/celestiaorg/celestia-app/pull/4796/commits/935a99cbb2ebcd08fe5494ac6dd99c1e2966b7ab

we can't use the dynamic one without also updating all the docker containers or something as they fail when calling `make mod`, so reverted to just using a hardcoded one for the 3 different go mods